server/src/main/java/com/skillmagnet/Enrolls/EnrollsRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.skillmagnet.Enrolls;
+
+import com.skillmagnet.Course.Course;
+import com.skillmagnet.User.User;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface EnrollsRepository extends JpaRepository<Enrolls, Long>{
+    List<Enrolls> findByEnrolledUser(User u);
+    Enrolls findByEnrolledCourseAndEnrolledUser(Course c, User u);
```

Comment:
Allows for finding specific enrollments easily, used in new user endpoint to update progress